pconf/__init__.py

```diff
@@ -1 +1,5 @@
 from .pconf import Pconf  # noqa: F401
+
+import logging
+
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
With the change from `warnings` to `logging` I'd skip this change. If we keep this then the default behavior changes to silently ignoring missing files instead of showing a warning.

Skipping this change still allows users to use their own loggers to format and possibly ignore these warnings.